docs/src/app/keystatic/layout.tsx

```diff
@@ -1,12 +1,5 @@
 import KeystaticApp from './keystatic';
 
 export default function RootLayout() {
-  return (
-    <html>
-      <head />
-      <body>
-        <KeystaticApp />
-      </body>
-    </html>
-  );
+  return <KeystaticApp />;
```

Comment:
The only thing to note here is that the Keystatic Admin UI will get the root layout with:

- metadata - I think this is fine, we can overwrite it in future if we need to
- Analytics - This might actually be good for when we set it up properly and encourage users to check our the keystatic admin for the keystatic page!

This is just a FYI, no changes requested